src/supervisor/metadata-extractor.ts

```diff
@@ -41,6 +42,10 @@ export function buildImageMetadata(
   const images: IWorkload[] = [];
   for (const containerStatus of containerStatuses) {
     if (!(containerStatus.name in containerNameToSpec)) {
+      logger.debug(
+        { containerName: containerStatus.name },
```

Comment:
Is it possible to add more context, otherwise how can we distinguish which container comes from where? For example, adding the workload name, type, and namespace!